### ISSUE TITLE
NextGen - W2/Defaults - Tab ordering incorrect when SS wage input default turned on

### ISSUE THREAD

**Issue by MHGray** [TaxwareSupport/Issue-Reports#40]
**Client Name:**  
Barrey

**Client ID:**  
87191

**Description:**  
![image](https://user-images.githubusercontent.com/8196735/48586434-c9c33080-e8e4-11e8-8850-5a92b03487cd.png)
When the above check box is checked, it doesn't have the same tab order that it does in Legacy with that checkbox checked.
Legacy w/ default:
![image](https://user-images.githubusercontent.com/8196735/48586417-bd3ed800-e8e4-11e8-962b-661d202d411f.png)
NextGen w/ default:
![image](https://user-images.githubusercontent.com/8196735/48586737-a0ef6b00-e8e5-11e8-8c6f-0e1dac070ed2.png)

**Comment by MyNameIsDonovan** [comment]
Fixed
